### ISSUE TITLE
Files crash while opening any folder

### ISSUE THREAD

**Issue by Doda94** [files-community/Files#6605]
### Description

Files crash when I open any folder
![KOZN84D8HF](https://user-images.githubusercontent.com/56204086/138613484-a7d47165-00d9-478b-9cc6-287b5597571e.gif)
.

### Expected behavior

App to not crash.

### Files Version

2.0.28.0

### Windows Version

Windows 11 21H2 22000.194

### Relevant Assets

[debug.log](https://github.com/files-community/Files/files/7406271/debug.log)
[debug_fulltrust.log](https://github.com/files-community/Files/files/7406272/debug_fulltrust.log)

**Comment by gave92** [comment]
@Doda94 for now if you wish to use the file tag feature you can try replacing the filetags.json file contents with:
```
{
  "FileTagList": [
    {
      "TagName": "Blue",
      "Uid": "3aab30c1-a9f6-4dbd-a3a8-c343b645b177",
      "ColorString": "#0072BD"
    },
    {
      "TagName": "Orange",
      "Uid": "2e1a1dda-688c-4966-a0eb-b28f2315a09f",
      "ColorString": "#D95319"
    },
    {
      "TagName": "Yellow",
      "Uid": "ebba15f5-12f4-4417-bab1-877ae182c171",
      "ColorString": "#EDB120"
    },
    {
      "TagName": "Green",
      "Uid": "5304fa1b-8153-4039-9595-c9dc66e924a1",
      "ColorString": "#77AC30"
    },
    {
      "TagName": "Azure",
      "Uid": "f3714fb0-cfe7-4f01-85c0-fbdda7abde99",
      "ColorString": "#4DBEEE"
    }
  ]
}
```

**Comment by gave92** [comment]
@d2dyno1 do you have any clue about why the above could be happening? The FileTags list setting gets serialized all wrong (it serializes only the field marked as JsonIgnore!)

**Comment by yaichenbaum** [comment]
This issue should be resolved with #7017.
